module/avl/avl.c

```diff
@@ -103,6 +103,7 @@
 #include <sys/debug.h>
 #include <sys/avl.h>
 #include <sys/cmn_err.h>
+#include <sys/zfs_context.h>
```

Comment:
Can we add a `sys/module.h` wrapper for this rather than including all of `sys/zfs_context.h`